frontend/components/DataSet/_styles.scss

```diff
@@ -1,7 +1,5 @@
 .data-set {
   font-size: $x-small;
-  max-width: min-content;
-  overflow: hidden;
```

Comment:
Had to remove this global fix as it caused 2 noticeable unreleased bugs, local fix incoming in 2/4/25 PR